app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1280,6 +1280,16 @@
                             <show_in_website>0</show_in_website>
                             <show_in_store>0</show_in_store>
                         </crate_admin_user_notification>
+                        <reprocess_image_quality translate="label comment">
+                            <label>Image Reprocess Quality</label>
+                            <comment>Optimum value is 85, higher value will increase file size. Set to 0 to turn off images reprocessing, which may cause security risks.</comment>
+                            <frontend_type>text</frontend_type>
```

Comment:
`<frontend_type>text</frontend_type>` not required as it's the default.

Reply:
Removed.